inventories/nos-social/inventory.yml.bkup_23-12-05_09:53

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
This file is a bit misleading.  It's created by our new-do-droplet  play when it creates a new server.  As part of the creation, we update our inventory file.  Because of how we have to do the update (essentially writing it again with the new values) I wanted to make sure we did a backup before the update, so that we can revert to the backup in case there's an issue and our inventory is not lost.  If everything goes to plan, then this backup file can be deleted.  

In other words, we shouldn't commit this file.  It can be deleted once you verified that the inventory update was as you expected.  We should probably add `inventory*.bkup_*` to our gitignore.  But this file, and the one below, can be deleted.